ensembl_rest_testing.conf

```diff
@@ -129,6 +129,7 @@ jsonp=1
 
 <Model::ga4gh::variantSet>
    ga_config = __path_to(t/test-genome-DBs/testdata/ga_data/ga_vcf_config.json)__
+   dir       = __path_to(t/test-genome-DBs/testdata/ga_data/)__
```

Comment:
There's a lot of redundancies in this part (i.e. Model::ga4gh) of the configuration file. All the subsections refer to the same ga_config file and their directories point to the same location.
I haven't seen the production configuration file, are all these to be deemed potentially different on production? If not, they could be shared up to the higher level.